src/components/Dashboard/Dashboard.jsx

```diff
@@ -14,19 +14,24 @@ import SearchBar from "material-ui-search-bar";
 // import LinearProgress from "@material-ui/core/LinearProgress";
 import { useStyleDashboard } from "./Widgets/Style";
 import FiltersFlat from "./Widgets/FiltersFlat";
-import { ProfileContext } from '../../profileContext' 
-
+import { ProfileContext } from "../../profileContext";
+import { db } from "../../firebase";
+import CircularProgress from "@material-ui/core/CircularProgress";
 
 export default function Dashboard() {
-  const [profile, ] = useContext(ProfileContext)
-
+  const [profile] = useContext(ProfileContext);
+  const [posts, setPosts] = useState();
   const [state, setState] = React.useState({
     mobileView: false,
     mediumView: false,
   });
 
   const { mobileView, mediumView } = state;
-
+  React.useEffect(() => {
```

Comment:
Again why are you using imported statements once and children again! Stick to a style, and I would say import things!

Reply:
Fixed 783ee049a3396f564058a194d966ebf9d3ca9f7e